### PR TITLE
Added SST Chodl

### LINKED ISSUE #1320: [NTR] SST chodl gabaergic interneuron
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
sst chodl GABAergic interneuron

**Synonyms**
[Broad] long-range GABAergic interneuron 

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
A sst GABAergic cortical interneuron that also expresses Chodl. These neurons are rare and correspond to the only known cortical interneurons with long-range projection. [PMID:15845086, PMID: 31209381, PMID: 33372656]

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
sst GABAergic cortical interneuron

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
cerebral cortex

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #1320

### CODE REVIEW

**Review by patrick-lloyd-ray — APPROVED** [review]
Looks good to me.

**Review by dosumis — COMMENTED** [review]
Looks good.  How confident are you in the cross-species nature of the term.  Is there anything on long range projection outside of mouse?

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -29154,6 +29155,18 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023120 "cochlear hair cell"
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15845086"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:31209381") Annotation(oboInOwl:hasDbXref "PMID:33372656") obo:IAO_0000115 obo:CL_4023121 "A sst GABAergic cortical interneuron that also expresses Chodl. These neurons are rare and correspond to the only known cortical interneurons with long-range projection.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023121 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_4023121 "long-range GABAergic interneuron")
+AnnotationAssertion(rdfs:label obo:CL_4023121 "sst chodl GABAergic cortical interneuron")
+SubClassOf(obo:CL_4023121 obo:CL_0000099)
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000005444))
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:29168]: Could potentially make this an equivalent Class def. I think you have N&S clauses.

**Comment by shawntanzk** [comment]
> How confident are you in the cross-species nature of the term. Is there anything on long range projection outside of mouse?

In terms of cross-species nature of this term, very confident that it exists cross-species. However, long-range projection experiments that show this seems to be all mouse (I guess projection studies are a lot more tricky with like human tissue). That being said, papers seem to talk as if it is generalizable - but in general rodent papers do this regardless of if it is.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:29168]: I did think about this, however, I didn't feel comfortable doing that - might be me being paranoid but cause SST Chodl does not have SST in NS-forest, but instead Nos1 and Chodl, I wasn't super confident in making that and equiv class

**Comment by shawntanzk** [comment]
messy merge from master - going to close and reopn for ease

**Pull request closed without merging**